Module_8/Q3.js

```diff
@@ -0,0 +1,22 @@
+//3. What will be printed to the console when the promise resolves and when it rejects?
+const testAsyncFunction = async () => {
+  return new Promise((resolve, reject) => {
+    if (Math.random() > 0.5) {
+      resolve('Test Resolve');
+    } else {
+      reject('Test Reject');
+    }
+  })
+    .catch((err) => {
```

Comment:
No need of catch block here as you are already handling it below on lin 18

Reply:
This code is provided by the LMS itself they have asked to check for the output